src/main.ts

```diff
@@ -47,13 +54,19 @@ const command: Command = {
 
 		options('c', {
 			alias: 'config',
-			describe: 'Specifies what configuration to test with: \'local\'(default), \'browserstack\', \'testingbot\', or \'saucelabs\'.',
+			describe: `Specifies what configuration to test with: 'local'(default), 'browserstack', 'testingbot', or 'saucelabs'.`,
```

Comment:
There is discussion in #30 and #21 about how this all a bit confusing.  I only changed the string formatting to be a string literal.  I suggest and additional verbiage changes we address in resolving either of those two issues.